FirebaseAuthUI/FUIPasswordSignInViewController.m

```diff
@@ -59,6 +59,11 @@ @implementation FUIPasswordSignInViewController {
       @brief The @c UIButton which handles forgot password action.
    */
   __weak IBOutlet UIButton *_forgotPasswordButton;
+
+  /** @var _onDismissCallback
+      @brief The callback to be executed during view controller dismissal.
+   */
+  FIRAuthDataResultCallback _onDismissCallback;
```

Comment:
A better name might be `onSuccessfulLoginCallback`, since this block will not be called if the controller is dismissed by exiting the login flow.